lib/card/theme.module.scss

```diff
@@ -1,105 +1,83 @@
 @import '../globals/theme';
-
 :local(.card) {
-  display: flex;
-  flex-direction: column;
-  box-sizing: border-box;
-  background: $original-white;
-  border-radius: 3px;
-  padding: 15px 20px;
-  margin: 5px;
-  font-family: Roboto, sans-serif;
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+    background: $original-white;
+    border-radius: 3px;
+    padding: 1.6%;
+    margin: 5px;
+    font-family: Roboto, sans-serif;
 }
+
 .expandedContent {
-  max-height: 0;
-  overflow: hidden;
-  &.expanded {
-    max-height: 2000px;
-    transition: max-height 1s ease;
-  }
-  &.collapsed {
     max-height: 0;
-    transition: max-height 0.5s ease;
-  }
+    overflow: hidden;
+    padding-top: 1%;
+    &.expanded {
+        max-height: 2000px;
+        transition: max-height 1s ease;
```

Comment:
- can we have same transition duration for both collapsing and expanding
- test between `linear` and `ease` timing function to check which looks better

Reply:
Move `transition` code to common CSS instead of putting in both `expanded` and `collapsed` states